docs/tutorials/excel-tutorial.md

```diff
@@ -770,7 +770,7 @@ In this final step of the tutorial, you'll open a dialog in your add-in, pass a
 
 3. Add the following markup to **popup.html**. Note:
 
-   - The page has a `<input>` where the user will enter their name and a button that will send the name to the page in the task pane where it will be displayed.
+   - The page has an `<input>` field where the user will enter their name, and a button that will send this name to the task pane page where it will display.
```

Comment:
I think "task pane" (not "task pane page") is more accurate, since it's embedded in Excel.
```suggestion
   - The page has an `<input>` field where the user will enter their name, and a button that will send this name to the task pane where it will display.
```